quick_run.sh

```
@@ -0,0 +1,137 @@
+    echo "❌ Ollama not running"
+    echo "   Start with: docker run -d -p 11434:11434 ollama/ollama"
+    echo "   Or use HuggingFace: --backend huggingface"
+    echo ""
+    echo "📝 Running in TEST mode (mock LLM)..."
+    MOCK_MODE=true
+else
+    echo "✅ Ollama is running"
+    MOCK_MODE=false
+
+    # Check if model exists
+    if ! docker exec -it $(docker ps -q -f ancestor=ollama/ollama) ollama list | grep -q "qwen2.5-coder"; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)